src/components/AdminSettings/SignalingServers.vue

```diff
@@ -184,6 +188,21 @@ export default {
 <style lang="scss" scoped>
 @import '../../assets/variables';
 
+.signaling-server {
+	h2 {
+		height: 44px;
+		display: flex;
+		align-items: center;
+	}
+
+	&__add-icon {
+		display: inline-block;
+		width: 44px;
+		height: 44px;
+		vertical-align: middle;
+	}
```

Comment:
Is this necessary? Should be handled by the `Button`, no?

Reply:
No, that's right. Not sure where it is coming from.